inductiva/molecules/scenarios/water_box/water_box.py

```diff
@@ -0,0 +1,145 @@
+"""Water box scenario."""
```

Comment:
I think here we should be more picky, since we have fluids scenarios as well.

Suggestion: "Water molecules box scenario".

Reply:
Maybe on the rest of the variables `water_box` this should also be added. Notice that, we have on the templates "templates/water_box/gromacs". If we have a CFD simulation with openfoam, for example, should it also go here, or this folder is just for MD?

Reply:
good point, should we had another layer of folders? Like, fluids, MD? Maybe it would be more organised.

Reply:
Keep it simple, just change the name.